src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -100,6 +101,17 @@ protected function prepare_items_query( $prepared_args = array(), $request = nul
 			add_filter( 'wp_allow_query_attachment_by_filename', '__return_true' );
 		}
 
+		// Map to proper WP_Query orderby param - this needs to happen AFTER parent class.
```

Comment:
```suggestion
		// Map to proper WP_Query orderby param.
```

Tiniest of tiny nits: feel free to ignore! Everything else in this function happens after the parent class, so I wasn't sure if this part of the comment was necessary?

Reply:
Ah no, that was me talking myself, and I committed it as a comment. Thanks for flagging!